src/makeRobot.js

```diff
@@ -39,6 +39,71 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  let x = 0;
+  let y = 0;
+
+  const robot = {
+
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+    get location() {
+      return `${name}: x=${x}, y=${y}`;
+    },
+    'coords': {
+      x: x,
+      y: y,
+    },
```

Comment:
```suggestion
    get info() {
      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
    },
    
    get location() {
      return `${name}: x=${x}, y=${y}`;
    },
    
    'coords': {
      x,
      y,
    },
```

But here (and other such places) better to add empty line

Reply:
And you can use [object key shorthand](https://attacomsian.com/blog/javascript-object-property-shorthand):

```
 'coords': {
      x,
      y,
    },
```